resources/guest_configs/microvm-kernel-ci-aarch64-4.14.config

```diff
@@ -717,39 +717,39 @@ CONFIG_IPV6_OPTIMISTIC_DAD=y
 # CONFIG_IPV6_TUNNEL is not set
 # CONFIG_IPV6_FOU is not set
 # CONFIG_IPV6_FOU_TUNNEL is not set
-CONFIG_IPV6_MULTIPLE_TABLES=y
-CONFIG_IPV6_SUBTREES=y
-CONFIG_IPV6_MROUTE=y
-CONFIG_IPV6_MROUTE_MULTIPLE_TABLES=y
-CONFIG_IPV6_PIMSM_V2=y
-CONFIG_IPV6_SEG6_LWTUNNEL=y
-CONFIG_IPV6_SEG6_HMAC=y
-CONFIG_NETLABEL=y
+CONFIG_IPV6_MULTIPLE_TABLES=n
+CONFIG_IPV6_SUBTREES=n
+CONFIG_IPV6_MROUTE=n
+CONFIG_IPV6_MROUTE_MULTIPLE_TABLES=n
+CONFIG_IPV6_PIMSM_V2=n
+CONFIG_IPV6_SEG6_LWTUNNEL=n
+CONFIG_IPV6_SEG6_HMAC=n
+CONFIG_NETLABEL=n
 CONFIG_NETWORK_SECMARK=y
 CONFIG_NET_PTP_CLASSIFY=y
 CONFIG_NETWORK_PHY_TIMESTAMPING=y
-CONFIG_NETFILTER=y
-CONFIG_NETFILTER_ADVANCED=y
+CONFIG_NETFILTER=n
+CONFIG_NETFILTER_ADVANCED=n
 
 #
 # Core Netfilter Configuration
 #
-CONFIG_NETFILTER_INGRESS=y
+CONFIG_NETFILTER_INGRESS=n
 # CONFIG_NETFILTER_NETLINK_ACCT is not set
 # CONFIG_NETFILTER_NETLINK_QUEUE is not set
 # CONFIG_NETFILTER_NETLINK_LOG is not set
-CONFIG_NF_CONNTRACK=y
-CONFIG_NF_LOG_COMMON=y
+CONFIG_NF_CONNTRACK=n
+CONFIG_NF_LOG_COMMON=n
 # CONFIG_NF_LOG_NETDEV is not set
-CONFIG_NF_CONNTRACK_MARK=y
-CONFIG_NF_CONNTRACK_SECMARK=y
-CONFIG_NF_CONNTRACK_PROCFS=y
-CONFIG_NF_CONNTRACK_EVENTS=y
-CONFIG_NF_CONNTRACK_TIMEOUT=y
-CONFIG_NF_CONNTRACK_TIMESTAMP=y
-CONFIG_NF_CT_PROTO_DCCP=y
-CONFIG_NF_CT_PROTO_SCTP=y
-CONFIG_NF_CT_PROTO_UDPLITE=y
+CONFIG_NF_CONNTRACK_MARK=n
+CONFIG_NF_CONNTRACK_SECMARK=n
+CONFIG_NF_CONNTRACK_PROCFS=n
+CONFIG_NF_CONNTRACK_EVENTS=n
+CONFIG_NF_CONNTRACK_TIMEOUT=n
+CONFIG_NF_CONNTRACK_TIMESTAMP=n
+CONFIG_NF_CT_PROTO_DCCP=n
+CONFIG_NF_CT_PROTO_SCTP=n
+CONFIG_NF_CT_PROTO_UDPLITE=n
```

Comment:
IIRC, some/all of these are needed for docker to run. Are we sure we want to remove those?

Reply:
Well we don't use them in any of our tests, so they are just dead weight. If we want to add tests that need them, we can re-enable them.